_topics/en/compact-block-filters.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Compact block filters
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP157
+  - BIP158
+  - Neutrino protocol
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+  - Lightweight Client Support
+  - P2P Network Protocol
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Compact block filters** are a condensed representation of the
+  contents of a block that allow wallets to determine whether the block
+  contains any transactions affecting the wallet.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A full node with a copy of a block and the data necessary to validate
+  that block uses [BIP158][] to create a Golomb-Rice Coded Set (GCS) of
+  data from that block (such as the prevout script from each input spent
+  in the block and the scriptPubKey for each output created in the
+  block).  The GCS (called a filter) is then distributed to wallets
+  (such as via the P2P protocol as described in [BIP157][]), allowing
+  them to search for any matches to each wallet's scripts.  If a match
+  is found, the wallet can then download the corresponding block to
+  access any relevant transactions.
+
+    The GCS mechanism guarantees that a properly-coded wallet will find
+    any transactions matching its scripts, but it may also find some
+    false-positive matches for which it will need to download and scan
+    the block despite the block not containing any transactions relevant
+    to the wallet.
+
+    The BIP157/158 protocol is sometimes called "Neutrino" after the
```

Comment:
Perhaps "is sometimes (incorrectly) called ..."

Reply:
Is it incorrect?  I don't like calling it neutrino so I'd like to be able to say with authority that it's incorrect, but the closest I've heard any of the authors of the proposal say to that effect is Osuntokun with:

> I'd wager that most developers reading this email right now are familiar with neutrino as a project. Many even routinely use "neutrino" to refer to BIP 157+158.  ([source](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-February/016656.html))

Reply:
I'm pretty sure it's incorrect. Laolu seems to be drawing a clear distinction between Neutrino (implementation) and BIP157/158 (protocol) in that email.

If you didn't want to use the word "incorrectly", I think I'd be happier with 'is sometimes referred to as "Neutrino" after...' or rather than 'is sometimes called "Neutrino" after...'. It might just be me, but the first one sounds a bit more like "Neutrino" is an accepted alternative name and the second sounds like it's simply reporting on what some people say. "Is referred to by some as "Neutrino" after..." would be taking it even further.

Reply:
As long as you're convinced, that means I can blame you when it all goes wrong.  :-)  I'll add "incorrectly".